tests/javascript_tests/test_color_view_controller.js

```diff
@@ -126,7 +126,7 @@ requirejs([
 
     test("Test ColorViewController.getColorList exceptions", function(){
       var five;
-      five = [0, 1, 2, 3, 4];
+      five = ['0', 'string1', 'string2', 'string3', 'string4'];
```

Comment:
No actual functionality was changed save that error throwing, so the tests made sure it still works the same. I can add helper function tests though.